projects.html

```diff
@@ -2,8 +2,273 @@
 <html>
 <head>
 	<title>Projects</title>
+	<style>
```

Comment:
(From here to the end of style) This complete style section needs to be written in a new file in `css/projects.css`. Writing css in html forms makes it cumbersome to read and destroys the code cleanliness. Shift this to that file.

Reply:
gotcha